app/packages/flashlight/src/index.ts

```diff
@@ -159,6 +159,12 @@ export default class Flashlight<K> {
     this.state = this.getEmptyState(this.config);
 
     this.showPixels();
+    this.element.dispatchEvent(
+      new CustomEvent("flashlight-refreshing", {
+        detail: "flashlight-refreshing",
```

Comment:
Since there's nothing in detail, you can omit it, in which case your predicate would look like this: 
```
() => true
```

Only pass something in detail if you have to have filtering rules for a given event type.